lib/init.ts

```diff
@@ -65,9 +50,6 @@ export function init(options: InitParams) {
 You can visit https://algolia.com/events/debugger instead.`);
   }
 
-  this._apiKey = options.apiKey;
-  this._appId = options.appId;
-
```

Comment:
I think you still need to map `options.apiKey` to `_apiKey` and the same for `_appId` in `setOptions()`.

Reply:
Aren't they supposed to be assigned by the `reduce` in `setOptions()`?

```js
Object.assign(
  target,
  Object.keys(options).reduce(
    (acc, key) => ({ ...acc, [`_${key}`]: options[key] }),
    {}
  )
);
```

Reply:
Oh you're right, I didn't notice this!